gnosis/eth/tests/utils.py

```diff
@@ -40,6 +40,36 @@ def just_test_if_mainnet_node() -> str:
     return mainnet_node_url
 
 
+def just_test_if_polygon_node() -> str:
+    polygon_node_url = os.environ.get("POLYGON_NODE")
```

Comment:
Nice, but please use `ETHEREUM_POLYGON_NODE` (as we use `ETHEREUM` as the _namespace_ for RPC related stuff)

Reply:
Updated @Uxio0